src/Kernel32/Kernel32.Helpers.cs

```diff
@@ -256,6 +256,172 @@ public static ArraySegment<byte> ReadFile(SafeObjectHandle hFile, int nNumberOfB
             return new ArraySegment<byte>(buffer, 0, bytesRead);
         }
 
+        /// <summary>
+        /// Retrieves information about the specified process
+        /// </summary>
+        /// <param name="hProcess">
+        /// A handle to the process. This handle must have the <see cref="ProcessAccess.PROCESS_SET_INFORMATION"/> access right.
+        /// For more information, see <a href="https://docs.microsoft.com/en-us/windows/desktop/ProcThread/process-security-and-access-rights">Process Security and Access Rights</a>
+        /// </param>
+        /// <param name="ProcessInformationClass">
+        /// A member of the <see cref="PROCESS_INFORMATION_CLASS"/> enumeration specifying the kind of information to retrieve.
+        /// </param>
+        /// <param name="ProcessInformation">
+        /// Pointer to an object to receive the type of information specified by the <paramref name="ProcessInformationClass"/> parameter
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is nonzero.
+        ///
+        /// If the function fails, the return value is zero.To get extended error information, call <see cref="Marshal.GetLastWin32Error"/>.
+        /// </returns>
+        /// <remarks>
+        /// In most situations, the size of payload in <paramref name="ProcessInformation"/> can be correctly inferred based on
+        /// the value of <paramref name="ProcessInformationClass"/>. This helper is intended to make it simple to call into
+        /// <see cref="GetProcessInformation(SafeObjectHandle, PROCESS_INFORMATION_CLASS, void*, uint)"/>.
+        /// </remarks>
+#pragma warning disable IDE1006 // Naming Styles
+        public static unsafe bool GetProcessInformation(
+            SafeObjectHandle hProcess,
+            PROCESS_INFORMATION_CLASS ProcessInformationClass,
+            void* ProcessInformation)
```

Comment:
IMO these methods are not consistent with our [guidance on helper methods](https://github.com/dotnet/pinvoke/blob/master/CONTRIBUTING.md#helper-methods). They're not doing memory management or doing something else that's unnatural in .NET that's hard to get right. It's just removing a single argument that to the caller is either trivial (using `sizeof` around a type they surely ought to know) or it'll fail with an exception because it can't do it. 
I suggest dropping them.